Source/WebCore/CMakeLists.txt

```diff
@@ -287,6 +287,14 @@ set(WebCore_NON_SVG_IDL_FILES
     Modules/cookie-consent/Navigator+CookieConsent.idl
     Modules/cookie-consent/RequestCookieConsentOptions.idl
 
+    Modules/cookie-store/CookieInit.idl
+    Modules/cookie-store/CookieListItem.idl
+    Modules/cookie-store/CookieSameSite.h
```

Comment:
This is the cause of the build failures. Should be "CookieSameSite.idl". I'll fix this in the next iteration.

Reply:
Fixed!